tfx/examples/custom_components/slack/setup.py

```diff
@@ -24,8 +24,8 @@ def _make_required_install_packages():
   # six, and protobuf) with TF.
   return [
       'slackclient>=2.0.0,<2.0.1',
-      'tfx>=0.14.0,<0.15.0',
-      'websocket-client>=0.56,<0.6',
```

Comment:
It'll be better to keep consistency. In this case, double digit as version number.

Reply:
I'm not sure what's the right upper bound, but I guess 0.60.0 was the intent(?). I added a path version here to be consistent with other lib versions.